tools/wptrunner/wptrunner/executors/protocol.py

```diff
@@ -1015,3 +1015,13 @@ def update_virtual_pressure_source(self, source_type, sample):
     @abstractmethod
     def remove_virtual_pressure_source(self, source_type):
         pass
+
+class ProtectedAudienceProtocolPart(ProtocolPart):
+    """Protocol part for Protected Audiences"""
```

Comment:
Nit: Protected Audience (without the trailing 's').

Reply:
Done.